src/main/java/uk/gov/hmcts/reform/sscs/service/HearingsService.java

```diff
@@ -289,6 +291,28 @@ private void updateCaseWithHearingResponseV2(HearingWrapper wrapper, HmcUpdateRe
 
     }
 
+    private void updateCaseDataWithHearingResponseCaseDetails(HmcUpdateResponse response, Long hearingRequestId, SscsCaseDetails sscsCaseDetails) {
```

Comment:
where is this called from? I can see any usage of it
Test coverage missing too?